pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -321,6 +323,9 @@ const (
 	ConditionTrue    ConditionStatus = "True"
 	ConditionFalse   ConditionStatus = "False"
 	ConditionUnknown ConditionStatus = "Unknown"
+
+	ConditionEnabled  ConditionStatus = "Enabled"
+	ConditionDisabled ConditionStatus = "Disabled"
```

Comment:
can't we use `ConditionTrue` and `ConditionFalse` instead of these?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1706/commits/5de4172c051d4d2c892fe1613b714953ba21407a